src/Controls/tests/DeviceTests/HandlerTestBase.Android.cs

```diff
@@ -99,11 +99,6 @@ Task RunWindowTest<THandler>(IWindow window, Func<THandler, Task> action)
 					{
 						aca.SetSupportActionBar(null);
 					}
-
-					// Ideally this wouldn't be needed but I haven't found the right platform
-					// component I can key into for knowing when the world can move on
-					await Task.Delay(1000);
-					fragmentManager.ExecutePendingTransactions();
```

Comment:
Was this meant to go?

Reply:
Yea this was in place before I grouped all the tests into collections. It made it randomly work better :-) Now that they all line up single file this isn't needed